.gitignore

```diff
@@ -0,0 +1,8 @@
+.eslintcache
+.env
+.env.test
+node_modules/
+coverage/
+build/
+husky.sh
+yarn.lock
```

Comment:
please also include `yarn-debug.log*, yarn-error.log*` to the `gitignore`

Reply:
done